### ISSUE TITLE
[EN][Spell][Hunter] Stable Pet -- Internal pet error

### ISSUE THREAD

**Issue by ghost** [Atlantiss/BugTracker#9461]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Stabling an Exotic pet while not in BM will cause an 'Internal Pet Error'.

**Current behaviour**: See description.

**Expected behaviour**: Putting an Exotic pet into the stable should complete, irrespective of the talents.

**Steps to reproduce the problem**:

1. Tame an Exotic pet.
2. Spec to a different build.
3. Stable the Exotic pet.

Alternatively:

1. Have an Exotic pet in your stable while in a different spec.
2. Move the Exotic pet while it still is in your stable, move it from stable slot to stable slot.

In short, without the Beast Mastery talent Exotic pets cannot be manipulated in any way, shape or form. Even if this manipulation does not involve Calling them or having them in your Call Pet slots.

**Revision**: 5482

**Comment by lamaATS** [comment]
Fixed in rev. 5493
